docs/reference/quantity_point.md

```diff
@@ -1,31 +1,422 @@
-  wouldn't make sense to _add_ two mile markers, but you can _subtract_ them and obtain a _distance_
-  (which is a [`Quantity`](./quantity.md), not a `QuantityPoint`).
+- `U` is the _unit_: a type representing the [unit of measure](./unit.md).
+- `R` is the _"rep"_, a term we borrow from the [`std::chrono`
+  library](https://en.cppreference.com/w/cpp/chrono/duration).  It's the underlying raw numeric type
+  which holds the wrapped value.
 
-!!! warning "TODO: this page is a stub"
+## Naming `QuantityPoint` in code
 
-We will provide a full-fledged reference for quantity points later.  For now, here are the basics.
+You can use both template parameters directly (for example, `QuantityPoint<Meters, int>`).  However,
+multiple template parameters can be awkward to read.  It can also give the parser some trouble when
+you're forced to use macros, as in googletest.  For that reason, we also provide "Rep-named
+aliases", so you can express the rep more precisely, and put the visual focus on the unit:
 
-1. `QuantityPoint<U, R>` stores a value of numeric type `R` (called the "Rep"), whose units are the
-   [unit type](./unit.md) `U`.
+| Rep-named alias | Equivalent to: |
+|-----------------|----------------|
+| `QuantityPointD<U>` | `QuantityPoint<U, double>` |
+| `QuantityPointF<U>` | `QuantityPoint<U, float>` |
+| `QuantityPointI<U>` | `QuantityPoint<U, int>` |
+| `QuantityPointU<U>` | `QuantityPoint<U, unsigned int>` |
+| `QuantityPointI32<U>` | `QuantityPoint<U, int32_t>` |
+| `QuantityPointU32<U>` | `QuantityPoint<U, uint32_t>` |
+| `QuantityPointI64<U>` | `QuantityPoint<U, int64_t>` |
+| `QuantityPointU64<U>` | `QuantityPoint<U, uint64_t>` |
 
-    - **Example:** `QuantityPoint<Meters, double>` represents an along-path point, whose distance is
-      measured in `Meters`, in a `double` variable.
+## Constructing `QuantityPoint` {#constructing}
 
-2. We provide "Rep-named aliases" for better ergonomics.
+There are two ways to construct a `QuantityPoint` object.  For concreteness, we'll use this
+skeleton to show the signatures.
 
-    - **Example:** `QuantityPointD<Meters>` is an alias for `QuantityPoint<Meters, double>`.
+```cpp
+template <typename Unit, typename Rep>
+class QuantityPoint {
 
-3. You cannot get values into or out of `QuantityPoint` without _explicitly naming the unit, at the
-   callsite_.  We do not have tutorials for this yet, but it works the same as `Quantity`.  In the
-   meantime, you can read [the `QuantityPoint` unit
-   tests](https://github.com/aurora-opensource/au/blob/main/au/quantity_point_test.cc) to see some
-   practical examples.
+   // A) Implicit constructor from another QuantityPoint
+   template <typename OtherUnit, typename OtherRep>
+   QuantityPoint(QuantityPoint<OtherUnit, OtherRep> other);
+
+   // B) Default constructor
+   QuantityPoint();
+};
+```
+
+However, the _preferred_ way to construct a `QuantityPoint` is actually neither of these.  It's the
+_quantity point maker_, which we describe next.
```

Comment:
Great callout, thanks.  The reason this section reads as it does is because I copied it from the `Quantity` doc and modified it.  I've cleaned it up here, and also made similar improvements in the quantity doc.  Specific cleanups include:

- Stating clearly up front that we prefer the quantity (point) makers.
- Splitting out the constructor signature expositions into their respective sections.

As for "default constructor", yes, that is the C++ term of art for a constructor that takes no arguments, and we don't expect our target users to find this confusing.